view/dispense.py

```diff
@@ -40,14 +40,15 @@ def by_project_index(pid: str) -> str | ResponseBase:
             return jsonify({'result': dispense})
 
         if data and data['casename'] == 'update':
-            # to do: support edit dispense
-            Expense.update_invoices(
-                expense_id=data['data']['_id'], invoices=data['data']['invoices'])
-            Expense.update_enable(
-                expense_id=data['data']['_id'], enable=data['data']['enable'])
-            result = Expense.update_status(
-                expense_id=data['data']['_id'], status=data['data']['status'])
-
-            return jsonify({'result': result})
+            return handle_update(data)
 
     return make_response({}, 404)
+
+def handle_update (data: dict[str, Any]) -> Response:
```

Comment:
這裡調整成 `-> ResponseBase`，上面的 L14 就不用加入 `Response`
```suggestion
def handle_update (data: dict[str, Any]) -> ResponseBase:
```

Reply:
done!